README.adoc

```diff
@@ -50,8 +65,38 @@ cd platforms/ios && pod install
 
 and then re build the application.
 
-==== Development and Contributing
-Looking to use the showcase for development purposes with the SDK please take some time to read the link:./CONTRIBUTING.md[General Contributing Guide]
+===== Work with Self-signed Certificate
```

Comment:
link to https://docs.aerogear.org/aerogear/latest/getting-started.html#using-self-signed-certificates-in-mobile-apps

Reply:
done